cmd/buildah/commit.go

```diff
@@ -130,20 +133,21 @@ func commitCmd(c *cobra.Command, args []string, iopts commitInputOptions) error
 		return errors.Wrapf(err, "error building system context")
 	}
 
-	dest, err := alltransports.ParseImageName(image)
-	if err != nil {
-		candidates, _, _, err := util.ResolveName(image, "", systemContext, store)
-		if err != nil {
-			return errors.Wrapf(err, "error parsing target image name %q", image)
-		}
-		if len(candidates) == 0 {
-			return errors.Errorf("error parsing target image name %q", image)
-		}
-		dest2, err2 := storage.Transport.ParseStoreReference(store, candidates[0])
-		if err2 != nil {
-			return errors.Wrapf(err, "error parsing target image name %q", image)
+	if image != "" {
```

Comment:
Is it possible for image to == ""?

Reply:
If we didn't get one passed as a command line argument, that's the value the variable has at this point.

Reply:
Ok I misread the code above.  I thought you were defaulting the image name to the container name if the user did not specify an image name.